airbyte-webapp/build.gradle

```diff
@@ -104,8 +105,15 @@ task copyNginx(type: Copy) {
     into "build/docker/bin/nginx"
 }
 
+task lint(type: NpmTask) {
+    dependsOn npmInstall
+    args = ['run', 'lint']
```

Comment:
We actually don't need to run `lint` completely, since Create React App already runs eslint as part of the `run build` (and thus we'd simply run it twice now). It should be enough just running the stylelint task individually via Gradle.

Reply:
Should I make an `npm run stylelint` or should I invoke it directly?